firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigFetchHttpClient.java

```diff
@@ -207,6 +215,15 @@ FetchResponse fetch(
     }
 
     ConfigContainer fetchedConfigs = extractConfigs(fetchResponse, currentTime);
+
+    long endTime = System.currentTimeMillis();
+    configLogger.logFetchEvent(
+        appId,
+        namespace,
+        instanceId,
+        /* timestampMillis= */ endTime,
```

Comment:
Minor: the event timestamp is specific to log logic, so I think it'd make more sense encapsulated in `ConfigLogger`. In other words, we happen to have a timestamp in this case, so it isn't tedious here, but it'd be tedious to require a timestamp every time we log an event.

Reply:
Fixed.